source/includes/documentation/_vue_js_elements.md

```diff
@@ -1145,39 +1145,134 @@ For examples of usage see:
 
 #### 5.12.2 Range Slider
 
-[Vuetify documentation page](https://vuetifyjs.com/en/components/range-sliders/)
+[Vuetify documentation page](https://v2.vuetifyjs.com/en/components/range-sliders/)
 
 - __Java__: _com.epam.jdi.light.vuetify.elements.complex.RangeSlider.java_
 
 ```java
-@Test
+//@FindBy(css = "#MinAndMaxRangeSlider .v-slider")
+@UI("#MinAndMaxRangeSlider .v-slider")
+public static RangeSlider minAndMaxRangeSlider;
+
+@Test(description = "Test checks min and max range slider's values: min(0-n), max(0-n)")
 public void minAndMaxRangeSliderTest() {
-    minAndMaxRangeSlider.slideHorizontalTo(11, 66);
-    minAndMaxRangeSlider.is().value(11, 66);
-    minAndMaxRangeSlider.slideHorizontalTo(-50, 90);
-    minAndMaxRangeSlider.is().value(-50, 90);
-}
+  minAndMaxRangeSlider.show();
+  minAndMaxRangeSlider.setValue(11, 66);
+  minAndMaxRangeSlider.has().value(11, 66)
+  .and().minValue(-50).and().maxValue(90);
+  minAndMaxRangeSlider.setValue(-50, 90);
+  minAndMaxRangeSlider.has().value(-50, 90);
+  }
 ```
 
-The Range Slider component is a better visualization of the number input. 
+The `v-slider` component is a better visualization of the number input. 
```

Comment:
should be 'v-range-slider' not to create mess. see API https://v2.vuetifyjs.com/en/api/v-range-slider/